eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -60,17 +60,18 @@ foreach ($res in $responses)
     }
     try {
       $commitDate = Get-GithubReferenceCommitDate -commitUrl $res.object.url -AuthToken $AuthToken
-      if ($commitDate -and ($commitDate -gt $LastCommitOlderThan)) {
-        LogDebug "The branch $branch last commit date $commitDate is newer than the date $LastCommitOlderThan. Skipping."
+      if (!$commitDate -or ($commitDate -gt $LastCommitOlderThan)) {
+        LogDebug "No last commit date or the branch $branch last commit date $commitDate is newer than the date $LastCommitOlderThan. Skipping."
         continue
       }
+      
+      LogDebug "Branch [ $branchName ] in repo [ $RepoId ] has no associated open Pull Request. Last commit date $commitDate is older than $LastCommitOlderThan. Deleting..."
     }
     catch {
       LogError "Get-GithubReferenceCommitDate failed with exception:`n$_"
       exit 1
     }
   } 
-  LogDebug "Branch [ $branchName ] in repo [ $RepoId ] has no associated open Pull Request. "
```

Comment:
I would keep this message because it is still useful for in the case we don't have a LastCommitOlderThan date